src/libs/polyfill/index.js

```diff
@@ -0,0 +1,18 @@
+import shouldPolyfill from './intl-polyfill';
```

Comment:
I'll let @techievivek comment on this before you make changes but I think this should be:

```
src/libs/IntlPolyfill/index.js
src/libs/IntlPolyfill/index.native.js
src/libs/IntlPolyfill/shouldPolyfill.js

```

and import the method in `setup/index.js` as:
```
import intlPolyfill from '../libs/IntlPolyfill';

...
...
...

intlPolyfill(); 

```

Reply:
I agree this makes it very much verbose.